### ISSUE TITLE
Improve the contrast/color perception

### ISSUE THREAD

**Issue by seemantk** [cfinst/cfinst.github.io#47]
During the call today, both Michael and Ronda observed that the lack of color contrast between the circular grid cells compromised the overall usability of the viz.  Thus, #46 addresses changing the cells to rectangular, and this one is for the color pallette
